cpp/dockerfiles/Dockerfile.wasm-linux-clang

```diff
@@ -9,4 +9,3 @@ RUN cmake --preset wasm -DUSE_TURBO=true && cmake --build --preset wasm
 FROM alpine:3.17
 COPY --from=builder /usr/src/aztec3-circuits/cpp/build-wasm/bin/aztec3-circuits.wasm /usr/src/aztec3-circuits/cpp/build-wasm/bin/aztec3-circuits.wasm
 COPY --from=builder /usr/src/aztec3-circuits/cpp/barretenberg/cpp/build-wasm/bin/primitives.wasm /usr/src/aztec3-circuits/cpp/barretenberg/cpp/build-wasm/bin/primitives.wasm
-COPY --from=builder /usr/src/aztec3-circuits/cpp/barretenberg/cpp/srs_db /usr/src/aztec3-circuits/cpp/barretenberg/cpp/srs_db
```

Comment:
This doesn't need to be there for non-assert dockerfiles. We only need the ignition scripts in "assert" dockerfiles that are then used to run tests.